com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ObjectStoreFileStorage.java

```diff
@@ -137,6 +137,22 @@ private void putBlobWithRetries(Blob blob, int retries) {
         }
     }
 
+    private Blob getBlobWithRetries(FileEntry fileEntry, int retries) {
+        Blob blob = null;
+        for (int i = 1; i <= retries; i++) {
+            blob = blobStore.getBlob(container, fileEntry.getId());

```

Comment:
Cloud we change this to  Blob blob = blobStore.getBlob(container, fileEntry.getId());

Reply:
done